lib/BitX.js

```diff
@@ -218,17 +218,20 @@ BitX.prototype.createFundingAddress = function (asset, callback) {
   this._request('POST', 'funding_address', {asset: asset}, callback)
 }
 
-BitX.prototype.getTransactions = function (asset, options, callback) {
+BitX.prototype.getTransactions = function (account_id, options, callback) {
```

Comment:
Please could you name the account id parameter using camel case so that it's consistent with the rest of the code like:

https://github.com/bausmeier/node-bitx/blob/54fd76a1f2a85af297f27dd079cd016f9a2dcb46/lib/BitX.js#L19

And please update the docs to match.